AdminServer/appscale/admin/__init__.py

```diff
@@ -140,6 +140,67 @@ def wait_for_delete(operation_id, http_port):
   operation.finish()
 
 
+class AppsHandler(BaseHandler):
+  """ Manages applications. """
+  def initialize(self, zk_client):
+    """ Defines required resources to handle requests.
+
+    Args:
+      zk_client: An KazooClient.
+    """
+    self.zk_client = zk_client
+
+  def get(self):
+    """ Retrieves projects.
+
+    Returns:
+      A list specifying the projects.
+    """
+    self.authenticate()
+
+    url = '/appscale/projects'
+    try:
+      apps = self.zk_client.get_children(url)
+    except NoNodeError:
+      message = 'Applications not found'
+      logging.warn('{}. URL: {}'.format(message, url))
+      raise CustomHTTPError(HTTPCodes.NOT_FOUND, message=message)
+
+    self.finish(json_encode(apps))
```

Comment:
`finish` is a little unusual to see outside the context of asynchronous handlers, but I I guess it works. The standard conventions for normal handlers is just `self.write`.